clients/src/main/java/org/apache/kafka/common/config/SaslConfigs.java

```diff
@@ -192,6 +192,12 @@ public class SaslConfigs {
             + " be inspected for the standard OAuth \"iss\" claim and if this value is set, the broker will match it exactly against what is in the JWT's \"iss\" claim. If there is no"
             + " match, the broker will reject the JWT and authentication will fail.";
 
+    public static final String SASL_OAUTHBEARER_HEADER_URLENCODE_ENABLE = "sasl.oauthbearer.header.urlencode.enable";
```

Comment:
Is there a KIP associated with this Jira ticket? Unfortunately, additions of configuration options required a KIP 😞

Reply:
It was added here: https://cwiki.apache.org/confluence/display/KAFKA/KIP-1025%3A+Optionally+URL-encode+clientID+and+clientSecret+in+authorization+header